STEER/STEER/AliGenerator.cxx

```diff
@@ -450,19 +450,37 @@ void AliGenerator:: SetHighWaterMark(Int_t nt)
   //
   if (fStack)
      fStack->SetHighWaterMark(nt);
-  else 
+  else
      gAlice->GetMCApp()->SetHighWaterMark(nt);
-   
+
 }
 void AliGenerator::FinishRun()
 {
     ;
 }
 
+///-------------------------------------------------------------------------
+Bool_t AliGenerator:: ApplyUserTrigger()
+{
+  // User Trigger
+  if(fSetUserTrig){
+    if(!fUserTrigger){
+      AliFatal("No external trigger defined!");
+      return kTRUE;
+    }
+    else{
+      if(!fUserTrigger(fStack)){return kFALSE;}
+      else{return kTRUE;}
+    }
+  }
+  else{return kTRUE;}
```

Comment:
you don't need the last "else", and neither the parenthesis after it (on line 476 nbut also 473), if it is only one line.

Reply:
Perfect! I remove the lase else and I re-commit